SimGeneral/MixingModule/python/mixObjects_cfi.py

```diff
@@ -221,6 +221,14 @@
 
 from SimCalorimetry.HGCalSimProducers.hgcalDigitizer_cfi import hgceeDigitizer, hgchefrontDigitizer
 
+from Configuration.Eras.Modifier_run2_GEM_2017_cff import run2_GEM_2017
+run2_GEM_2017.toModify( theMixObjects,
+    mixSH = dict(
+        input = theMixObjects.mixSH.input + [ cms.InputTag("g4SimHits","MuonGEMHits") ],
+        subdets = theMixObjects.mixSH.subdets + [ 'MuonGEMHits' ],
+        crossingFrames = theMixObjects.mixSH.crossingFrames + [ 'MuonGEMHits' ]
```

Comment:
@calabria 
do you have an explanation to what is happening with the mixing module setup in step3?
Please check on your side (with just the Run2_2017 era, no GEM_MCTest addition).